lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -4686,6 +4952,64 @@ impl FunctionCodeGenerator<CodegenError> for X64FunctionCode {
                 }
             }
 
+            Operator::I64TruncSSatF32 => {
+                let loc =
+                    get_location_released(a, &mut self.machine, self.value_stack.pop().unwrap());
+                let ret = self.machine.acquire_locations(
+                    a,
+                    &[(WpType::I64, MachineValue::WasmStack(self.value_stack.len()))],
+                    false,
+                )[0];
+                self.value_stack.push(ret);
+
+                let tmp_out = self.machine.acquire_temp_gpr().unwrap();
+                let tmp_in = self.machine.acquire_temp_xmm().unwrap();
+
+                Self::emit_relaxed_binop(
+                    a,
+                    &mut self.machine,
+                    Assembler::emit_mov,
+                    Size::S32,
+                    loc,
+                    Location::XMM(tmp_in),
+                );
+                Self::emit_f32_int_conv_check_sat(
+                    a,
+                    &mut self.machine,
+                    tmp_in,
+                    -9223373136366403584.0,
+                    9223372036854775808.0,
```

Comment:
Same here

Reply:
`GEF32_LT_I64_MIN` and `LEF32_GT_I64_MAX`.